ui/perfherder/graphs/LegendCard.jsx

```diff
@@ -162,6 +164,9 @@ const LegendCard = ({
 
           {series.name}
         </Button>
+        <div className="small">
+          {framework === 'talos' && <a href={getTalosDocsURL(suite)}>(docs)</a>}
```

Comment:
It'd be more consistent to provide the link right after the series name (immediately above).